### ISSUE TITLE
Dreambooth Training with Dreamlike Photoreal 2.0 on Google Colab

### ISSUE THREAD

**Issue by VillaesterModerneMedien** [TheLastBen/fast-stable-diffusion#2192]
Hello,

I'm trying to train with Dreamlike Photoral 2.0 and I'm also getting the error:

 raise ValueError("Attempting to unscale FP16 gradients.")
ValueError: Attempting to unscale FP16 gradients.

I have uploaded both the ckpt and safetensors and adjusted the path. I also removed the huggingface path as a test. But I just can't get it to work with this model. What am I doing wrong?

Attached is a screenshot with my settings

<img width="1464" alt="image" src="https://github.com/TheLastBen/fast-stable-diffusion/assets/77295564/c9c5f20e-210d-49bf-a64f-99f64d891535">

Thank you very much.

**Comment by TheLastBen** [comment]
the diffusers version of that model is in fp16 which is not suitable for training, use the ckpt/safetensors link instead

**Comment by TheLastBen** [comment]
@shluggy yes, use the link https://huggingface.co/dreamlike-art/dreamlike-photoreal-2.0/resolve/main/dreamlike-photoreal-2.0.safetensors in the input box MODEL_LINK and the notebook will use it as a base model

**Comment by eddmartinalfp** [comment]
I also tried to use dermlike-photoreal-2.0 and got this error
![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/57398438/fec5781e-7088-4c54-8a55-5691c5eb407f)
how can I solve that?

**Comment by TheLastBen** [comment]
@eddmartinalfp 
use the link https://huggingface.co/dreamlike-art/dreamlike-photoreal-2.0/resolve/main/dreamlike-photoreal-2.0.safetensors in the input box MODEL_LINK and the notebook will use it as a base model, tick the box "safetensors"
